components/prism-falcon.js

```diff
@@ -0,0 +1,26 @@
+Prism.languages.falcon = Prism.languages.extend('javascript', {
+	'number': [
+		Prism.languages.javascript.number,
+		{
+			pattern: /\b(?:api)\b/
```

Comment:
1. Why is `api` a number?
2. You don't need the group.
3. You don't need to create an object with a `pattern` property. You can just inline the regular expression.

```diff
- {
-     pattern: /\b(?:api)\b/
- }
+ /\bapi\b/
```